### ISSUE TITLE
Diagram: Allow ports to freely float around a box

### ISSUE THREAD

**Issue by bkolb** [JetBrains/MPS-extensions#284]
Imagine a language where you connect 2 network switches. Ports represent the different outlets of a switch. the lines are the network cables. Currently it is not possible to let it to the layouter to decide where the port would fit best. This results in a pretty bad auto layout.

**Comment by slisson** [comment]
The layouter is already able to change the order of the ports, but they will stay on the same side of the box.

**Comment by bkolb** [comment]
yes. Is it possible to also let the ports change the side? see #285 for an example

**Comment by arimer** [comment]
@alexanderpann, @slisson  do you know if there was any change trageting this issue?

**Comment by kbirken** [comment]
Removed myself as assignee, never worked on that topic.
